omnibus/omnibus.rb

```diff
@@ -35,7 +35,7 @@
 
 # Enable S3 asset caching
 # ------------------------------
-use_s3_caching true
+use_s3_caching false
```

Comment:
Should this be configurable?

Reply:
Added an environment variable that can be used to toggle this config value